frontend/src/api.js

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+
+const API_BASE_URL = "http://localhost:8000/api";
```

Comment:
In the future we should have a config file where we read all these URL so it can be configured in one place